src/main/java/dk/xam/jbang/Main.java

```diff
@@ -260,6 +260,42 @@ public Integer call() throws IOException {
 		}
 	}
 
+	static boolean checkVersions() {
+		String version = System.getProperty("java.version");
+		if (version.startsWith("1.")) {
+			version = version.substring(2, 3);
+		} else {
+			int dot = version.indexOf(".");
+			if (dot != -1) {
+				version = version.substring(0, dot);
+			}
+		}
+		int javaVersion = Integer.parseInt(version);
+		int javacVersion = 0;
+		try {
+			Process p = Runtime.getRuntime().exec("javac -version");
+			StringBuilder output = new StringBuilder();
+			BufferedReader reader = new BufferedReader(new InputStreamReader(p.getInputStream()));
+			String line;
+			while ((line = reader.readLine()) != null) {
+				output.append(line + "\n");
+			}
+			String[] s = output.toString().split(" ");
+			if (s[1].startsWith("1.")) {
+				s[1] = s[1].substring(2, 3);
+			} else {
+				int dot = s[1].indexOf(".");
```

Comment:
in the checks you can see the following stacktrace:

```
java.lang.ArrayIndexOutOfBoundsException: 1
	at dk.xam.jbang.Main.checkVersions(Main.java:287)
	at dk.xam.jbang.Main.build(Main.java:323)
```

so this check you are doing is not working on whatever version string github action linux is running with.